website/pages/docs/templates/user-variables.mdx

```diff
@@ -191,10 +191,25 @@ and detailed documentation for usage of each of those variables can be found
 ## AWS Secrets Manager Variables
 
 Secrets can be read from [AWS Secrets Manager](https://aws.amazon.com/secrets-manager/)
-and used within your template as user variables. the `aws_secretsmanager` function is
+and used within your template as user variables. The `aws_secretsmanager` function is
 available _only_ within the default value of a user variable, allowing you to default
 a user variable to an AWS Secrets Manager secret.
 
+
+### Plaintext Secrets
+```json
+{
+  "variables": {
+    "password": "{{ aws_secretsmanager `globalpassword` }}"
```

Comment:
This is a function we need to add to the the [HCL2: implementation list](https://github.com/hashicorp/packer/issues/9176)!